### ISSUE TITLE
[193] %1 is not a valid Win32 application.

### ISSUE THREAD

**Issue by Xemiru** [libgdx/packr#66]
Using a locally-built version of packr at commit https://github.com/libgdx/packr/commit/08a74d44ab6c33980741aace507d69fdaf6ca9da, since you guys don't seem to update the main download link. This seems to break SLF4J logging during building.

The program compiles fine, but the executable does nothing other than provide me with the title message.
```
Loading JVM runtime library ...
Error code [193]: %1 is not a valid Win32 application.
Error: failed to load VM runtime library!
```

Using OpenJDK 1.8.0_91-3 from https://github.com/ojdkbuild/ojdkbuild as the bundled JRE on a Windows 7 x64 system, although the same occurs when using an official Oracle JRE.

```batch
java -jar packr.jar --platform "windows32" --jdk "C:\Users\Xemiru\Desktop\packr\jre\win32-1.8.0_91.zip" --output "build-win32" -- config.json
@PAUSE
```

```json
{
	"platform": "windows32",
	"executable": "undertailor",
	"classpath": [
		"undertailor.jar"
	],
	"mainclass": "me.scarlet.undertailor.desktop.DesktopLauncher",
	"vmargs": ["Xms512M", "Xmx1536M"],
	"minimizejre": "oraclejre8",
	"jdk": "jre/win32-1.8.0_91.zip",
	"output": "build-win32",
	"verbose": true
}
```
Configuration options above.

**Comment by Xemiru** [comment]
Packaging with Win64 seems to work initially, until it decides to crash due to not finding a .dll the minimized JRE clearly has.

```
Loading JVM runtime library ...
Passing VM options ...
  # -Xms512M
  # -Xmx1536M
Creating Java VM ...

The program can't start because ojdkbuild_zlib.dll is missing from your computer. Try reinstalling the program to fix this problem.
```

**Comment by code-disaster** [comment]
The download links at https://github.com/ojdkbuild/ojdkbuild suggest that those are 64-bit builds of the JDK. The "windows32" executable of packr won't be able to load jvm.dll.

You either need to do a "windows64" build, or use a 32-bit JVM.

**Comment by Xemiru** [comment]
@code-disaster OpenJDK refuses to load my jar file anyway for whatever reason, so I can't really use it. I swapped out OpenJDK with a 32bit version of the Oracle JRE. Even without minifying, it still resolves to the same error.

**Comment by code-disaster** [comment]
Lets try to tackle one issue at a time. The "oracle8jre" minimize profile isn't intended to be used (and has never been tested) with OpenJDK, so I wouldn't be surprised if it messes stuff up. You should try to get it running without minimization first.

In case your code does not require Java 8, you can try with a proven JDK version, e.g. OpenJDK 7u45.

In your build, please check if there's ```.\jre\bin\server\jvm.dll``` present in the first place. Then, for a simple test, you can copy packr-windows.exe and packr-windows-64.exe from ```[packr]\src\main\resources\``` next to your ```undertailor.exe```, and try running both.

**Comment by Xemiru** [comment]
I ran the program without minification to test the Oracle JRE. It still kills the program with the original message. The Win64 version, however, works fine, with `oracle8jre` minification and a 64bit Oracle JRE.

My code has a _lot_ of Java 8 things, so it wouldn't work on anything else.

The jvm.dll is present in both the 64bit and 32bit versions, the 32bit version is the only thing crashing with the original message.

Using the original executables found within the `packr.jar` file produces the same result, where the 32bit version cries the same error and the 64bit version successfully executes the JAR file (albeit having another issue that I opened up in #67).
